build.gradle

```diff
@@ -38,18 +38,6 @@ allprojects {
 }
 
 subprojects {
-    configurations.configureEach {
-        resolutionStrategy.eachDependency { DependencyResolveDetails details ->
-            def group = details.requested.group
-            def module = details.requested.module.name
-
-            if (group == 'org.jetbrains.kotlin' && module == 'kotlin-stdlib-jdk7') {
-                // Force kotlin-stdlib module version
-                details.useTarget Libs.Kotlin.stdlib
-            }
-        }
-    }
-
```

Comment:
this was causing: ` Problems reading data from Binary store in /Users/runner/.gradle/.tmp/gradle17138586918115845078.bin offset 27715 exists? true`